diff --git a/sonic-silver-theme/sonic-silver-theme.spec b/sonic-silver-theme/sonic-silver-theme.spec
--- a/sonic-silver-theme/sonic-silver-theme.spec
+++ b/sonic-silver-theme/sonic-silver-theme.spec
@@ -4,0 +5 @@
+Patch0: sonic-silver-theme.source.patch
@@ -48,3 +49,0 @@
-BuildRequires:  cmake(Qt6ExamplesAssetDownloaderPrivate)
-BuildRequires:  pkgconfig(Qt6QmlAssetDownloader)
-BuildRequires:  qt6-qtbase-theme-gtk3
@@ -56 +54 @@
-%autosetup -p1
+%autosetup -p1 -n silver-theme-%{version}
@@ -60 +58,0 @@
-%if %[(0%{?rhel} >= 10)]
@@ -65 +62,4 @@
-%endif
+%cmake_build
+
+%install
+%cmake_install
