diff --git a/sonic-silver-theme/sonic-silver-theme.spec b/sonic-silver-theme/sonic-silver-theme.spec
--- a/sonic-silver-theme/sonic-silver-theme.spec
+++ b/sonic-silver-theme/sonic-silver-theme.spec
@@ -4,0 +5 @@
+Patch0: sonic-silver-theme.source.patch
@@ -48,3 +49,0 @@
-BuildRequires:  cmake(Qt6ExamplesAssetDownloaderPrivate)
-BuildRequires:  pkgconfig(Qt6QmlAssetDownloader)
-BuildRequires:  qt6-qtbase-theme-gtk3
@@ -56 +54 @@
-%autosetup -p1
+%autosetup -p1 -n silver-theme-%{version}
@@ -60 +58,0 @@
-%if %[(0%{?rhel} >= 10)]
@@ -65 +62,4 @@
-%endif
+%cmake_build
+
+%install
+%cmake_install
@@ -70,20 +70,22 @@
-%files
-%{_bindir}/%{name}-settings
-%{_libdir}/cmake/Klassy
-%{_libdir}/libklassycommon*
-%{_libdir}/qt5/plugins/styles/klassy5.so
-%{_libdir}/qt6/plugins/kstyle_config/klassystyleconfig.so
-%{_libdir}/qt6/plugins/org.kde.kdecoration3.kcm/kcm_klassydecoration.so
-%{_libdir}/qt6/plugins/org.kde.kdecoration3.kcm/klassydecoration
-%{_libdir}/qt6/plugins/org.kde.kdecoration3/org.kde.klassy.so
-%{_libdir}/qt6/plugins/styles/klassy6.so
-%{_iconsdir}/klassy-dark
-%{_iconsdir}/klassy
-%{_iconsdir}/hicolor/scalable/apps/klassy-settings.svgz
-%{_datadir}/kstyle/themes/%{name}.themerc
-%{_datadir}/plasma/desktoptheme/klassy
-%{_datadir}/plasma/layout-templates/*
-%{_datadir}/plasma/look-and-feel/*
-%{_datadir}/applications/kcm_klassydecoration.desktop
-%{_datadir}/applications/klassy*
-%{_datadir}/color-schemes/Klassy*
+%files
+%{_bindir}/silver-settings
+%{_libdir}/cmake/Silver
+%{_libdir}/libsilvercommon6.so.*
+%{_libdir}/qt6/plugins/kstyle_config/silverstyleconfig.so
+%{_libdir}/qt6/plugins/org.kde.kdecoration3.kcm/kcm_silverdecoration.so
+%{_libdir}/qt6/plugins/org.kde.kdecoration3.kcm/silverdecoration
+%{_libdir}/qt6/plugins/org.kde.kdecoration3/org.kde.silver.so
+%{_libdir}/qt6/plugins/styles/silver6.so
+%{_iconsdir}/silver
+%{_iconsdir}/silver-dark
+%{_iconsdir}/hicolor/scalable/apps/silver-settings.svgz
+%{_datadir}/kstyle/themes/silver.themerc
+%{_datadir}/plasma/desktoptheme/silver-light
+%{_datadir}/plasma/desktoptheme/silver-dark
+%{_datadir}/plasma/layout-templates/org.kde.silver*
+%{_datadir}/plasma/look-and-feel/org.kde.silver*
+%{_datadir}/applications/kcm_silverdecoration.desktop
+%{_datadir}/applications/silver-settings.desktop
+%{_datadir}/color-schemes/Silver*
+%{_datadir}/color-schemes/OpalFruits*
+%{_datadir}/locale/*/LC_MESSAGES/silver_*.mo
